libs/spark/src/NavBarItem.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Button,
+  ButtonBase,
+  ButtonBaseProps as MuiButtonBaseProps,
+} from '@material-ui/core';
+import { ButtonProps } from './Button';
+import { theme } from './styles';
+// import clsx from 'clsx';
+// import { capitalize } from './utils';
+
+export type NavBarItemProps = ButtonProps;
```

Comment:
`interface` not `type`, extend Mui's `ButtonProps`, omit `variant`, `color`. see CONTRIBUTING.md for a template on this

Reply:
What's the harm of doing a type here? I'm not getting an error when using a type when using Omit

```
export type NavBarItemProps = Omit<ButtonProps, 'variant' | 'color' | 'size'>;
// export interface NavBarItemProps
//   extends Omit<ButtonProps, 'variant' | 'color' | 'size'> {}
```